src/main/java/frc/robot/RobotContainer.java

```diff
@@ -165,12 +162,18 @@ private void configureButtonBindings() {
 
     // Shooter
     if (shooterSubsystem != null) {
+      shooterPrime = new ShooterPrime(shooterSubsystem, limelightSubsystem, CDSSubsystem);
       buttons[Constants.Xbutton].whenPressed(shooterPrime);
       buttons[Constants.upPOV].whenPressed(new InstantCommand(shooterSubsystem::cycleAimModeUp, shooterSubsystem));
       buttons[Constants.downPOV].whenPressed(new InstantCommand(shooterSubsystem::cycleAimModeDown, shooterSubsystem));
     }
 
-    if (CDSSubsystem != null) {
+    if (CDSSubsystem != null && shooterSubsystem != null) {
+      CDSForwardCommand = new CDSForwardCommand(CDSSubsystem, shooterSubsystem);
+      CDSReverseCommand = new CDSReverseCommand(CDSSubsystem, shooterSubsystem);
+      CDSSubsystem.getAllianceColor();
+      CDSSubsystem.senseColor();
```

Comment:
This doesn't do anything since the return value is unused.

Reply:
The method logs to SmartDashboard since all we wanted to do was test at the time. We'll work on adding functionality now.